code/modules/mining/lavaland/weather.dm

```diff
@@ -0,0 +1,110 @@
+var/global/datum/weather/activeWeather
```

Comment:
is there any need for this to be a single effect?
what if we want ash storms AND lava tides at once?